viewing_party/party.py

```diff
@@ -1,23 +1,150 @@
 # ------------- WAVE 1 --------------------
 
+from enum import unique
+
+
 def create_movie(title, genre, rating):
-    pass
+    my_dict = {}
 
-# -----------------------------------------
+    if not bool(title) or not bool(genre) or not bool(rating):
+        return None
+    else:
+        my_dict["title"] = title
+        my_dict["genre"] = genre
+        my_dict["rating"] = rating
+        
+    return my_dict
+
+def add_to_watched(user_data, movie):
+    watched_movies_list = user_data.get('watched')
+    watched_movies_list.append(movie)
+    user_data["watched"] = watched_movies_list
+    
+    return user_data

```

Comment:
Check out what happens when you remove line 21. If you run it, the tests still pass, so what's up with that? 

Well, lists are passed by reference, meaning that whether we append using the variable `user_data["watched"]` or `watched_movies_list`, the same list is being appended to. We don't have to re-assign `user_data["watched"]` with the value of `watched_movies_list` because it is referencing the same list that was changed.

Otherwise, this implementation works fine! But it is a little wordy. We could do this in 1 or 2 lines without creating new variables.

```suggestion
def add_to_watched(user_data, movie):
    user_data["watched"].append(movie)
    return user_data

```